pages/04.applications/99.wishlist/apps_wishlist.md

```diff
@@ -164,6 +164,7 @@ You can [contribute to this list by adding something you'd like to be packaged](
 | [Lessy](https://lessy.io) |  | [Upstream](https://github.com/lessy-community/lessy) |  |
 | [Librarian](https://lbry.bcow.xyz/) | An alternative frontend for LBRY/Odysee | [Upstream](https://codeberg.org/librarian/librarian) |  |
 | [LibreTranslate](https://libretranslate.com/) | Translation API | [Upstream](https://github.com/uav4geo/LibreTranslate) | [Package Draft](https://github.com/YunoHost-Apps/libretranslate_ynh) |
+| [Lichen](https://git.sensorstation.co/lichen.git) |  | [Upstream](https://git.sensorstation.co/lichen.git) |  |
```

Comment:
Please don't copypasta the code repo link in the first column ... this is meant to be exclusively for proper website

Reply:
Sorry, there is no website available.

Reply:
```suggestion
| Lichen |  | [Upstream](https://git.sensorstation.co/lichen.git) |  |
```